static/satsale.js

```diff
@@ -120,3 +120,17 @@ function copyTextFromElement(elementID) {
   copyText(elementText); //use the copyText function below
   alert("Copied address:" + elementText)
 }
+
+function display_currencies() {
+    const fs = require('fs');
+    const toml = require('toml');
+    const config = toml.parse(fs.readFileSync('../config.toml', 'utf-8'));
```

Comment:
This will not work, as this is client side JS, not server side.

Reply:
Have removed this!